src/components/FormModal/FormModal.js

```diff
@@ -62,10 +68,59 @@ export default function FormModal({ onCancelAdding, id, name }) {
     setOpeningDate(date);
   }
 
+  const routineData = getDataFromLocalStorage();
+
+  const sameProduct = routineData.filter((sameProduct) => {
+    return sameProduct.id === id;
+  });
+
+  function sendNameToIndex(name) {
+    const i = getIndexForWeekday(name);
+    setIndexWeekday(i);
+  }
+
+  const sameProductMorning = sameProduct.filter((productObject) => {
+    const morning = productObject.days[indexWeekday].morning === true;
+    return morning;
```

Comment:
In my opinion you can save a line of code. You don't need store the boolean in `morning` and then returning it. You can do it like in line 74

```suggestion
    return productObject.days[indexWeekday].morning === true;
```